### ISSUE TITLE
Doesn't work with Julia 0.3.0-dev

### ISSUE THREAD

**Issue by gitfoxi** [forio/julia-studio#212]
I have a recent build of Julia on OSX. When I point the path to Julia binaries at it there's several deprecation warnings on startup. Then when I try something at the command line it hangs forever. I have Julia Studio 0.4.4 binary install.

```

WARNING: take is deprecated, use take! instead.WARNING: 
put is deprecated, use put! instead. in 
event_loop in  at put/Applications/JuliaStudio.app/Contents/Resources/juliet/src/modules/event/event.jl at :deprecated.jl69:
8 in 
run in  at new_event/Applications/JuliaStudio.app/Contents/Resources/juliet/src/juliet-engine.jl at :/Applications/JuliaStudio.app/Contents/Resources/juliet/src/modules/event/event.jl48:
45 in 
include_from_node1 in  at process_incomingloading.jl at :/Applications/JuliaStudio.app/Contents/Resources/juliet/src/modules/network/network.jl120:
60
 in anonymous at task.jl:110

julia> 
WARNING: take is deprecated, use take! instead.
julia> 

 in handle_pending_events at /Applications/JuliaStudio.app/Contents/Resources/juliet/src/modules/event/event.jl:89
 in update at /Applications/JuliaStudio.app/Contents/Resources/juliet/src/modules/event/event.jl:59
 in event_loop at /Applications/JuliaStudio.app/Contents/Resources/juliet/src/modules/event/event.jl:72
 in run at /Applications/JuliaStudio.app/Contents/Resources/juliet/src/juliet-engine.jl:48
 in include_from_node1 at loading.jl:120
WARNING: take is deprecated, use take! instead.
 in event_loop at /Applications/JuliaStudio.app/Contents/Resources/juliet/src/modules/event/event.jl:69
 in run
julia> 
 at /Applications/JuliaStudio.app/Contents/Resources/juliet/src/juliet-engine.jl:48
 in include_from_node1 at loading.jl:120

julia> VERSION
```

**Comment by WestleyArgentum** [comment]
I've started a branch here: https://github.com/forio/julia-studio/tree/julia-0.3-compatibility

We'll try to address problems there

**Comment by keesvp** [comment]
depredations are fixed by Wes.
console input fixed too.
let's wait for more to show up.

**Comment by cwliu007** [comment]
Hi all,
I'm new to Julia and Julia studio. I have tried the Julia studio downloaded on its website. But, it is bundled with Julia 0.2 only. A multivariate normal random number sampling of package Distributions required Julia 0.3+. So, I tried to figure out how to compile Julia 0.3 and Studio by the webpage https://github.com/forio/julia-studio/blob/julia-0.3-compatibility/windows-build.md. However, I am confused by many unfamiliar terms like Qt SDK, branch of Qt, master branch, and so on. I am totally stuck. I am asking if any other instruction providing more specific step-by-step for newbie like me? 

Thank you all.

**Comment by npalmius** [comment]
Hi,

Just wondering if there is an expected release date for an official build of Julia Studio bundled with Julia 0.3? I've only just started using Julia and I'm loving Julia Studio, but I keep hearing that there are many improvements in Julia 0.3.

**Comment by WestleyArgentum** [comment]
Hey @npalmius, sorry this is so frustrating.

I'm speaking for myself here, and it's entirely possible there are things I don't know. But the reality -as I see it- is that @keesvp and I were the only developers at Forio working on Julia Studio, and neither of us is there now. Forio may put together a bundle that includes 0.3, but it's very unlikely they will put serious resources towards Julia Studio.

But that shouldn't dampen your enthusiasm for Julia! The language can be downloaded as a bundle here: http://julialang.org/downloads/ and you can write code in any editor! If you don't have a favorite, I strongly suggest trying [Sublime Text](http://www.sublimetext.com/3). There are syntax highlighting bundles for most editors, and there are even projects to integrate the REPL and IJulia into Sublime.

What's more, the built-in Julia REPL underwent an amazing overhaul in 0.3, and is leaps and bounds beyond any other I've used. I seriously recommend glancing over the [Getting Started docs](http://julia.readthedocs.org/en/latest/manual/getting-started/), installing a few packages, and poking around in the vanilla REPL :)

**Comment by npalmius** [comment]
Hey @WestleyArgentum, That's disappointing to hear but thank you for taking the time to update us on the project. I really like Julia Studio but I will investigate other options. I may also attempt a build from source as Julia 0.3 compatibility seems to have been merged to master now. Thanks again.

**Comment by WestleyArgentum** [comment]
@npalmius Absolutely! I believe if you build from master you'll be able to point at a Julia 0.3 bundle and be off-and-running. Sorry to be such a downer!
